lib/api.js

```diff
@@ -152,22 +152,15 @@ function shellScriptWorker(script = "worker.sh", options = {}) {
 
 async function buildTransformers(renditionCallback, options={}) {
     const transformers = [];
-
-    if (!options.manifests) {
+    if (!options.manifest) {
         return transformers;
     }
-
-    const manifests = options.manifests;
-    console.log('Manifest array', manifests);
-
-    manifests.forEach(oneManifest => {
-        const oneTransformer = new WorkerCallbackTransformer(renditionCallback, oneManifest);
-        debug(`Created a transformer named ${oneTransformer.name} with manifest:`);
-        debug(oneManifest);
-        transformers.push(oneTransformer);
-    });
-    return transformers.concat(options.transformerCatalog);
     
+    debug('Manifest', options.manifest);
```

Comment:
Add the transformer name to the log line, so it's easy to debug (in case it crashes during the constructor call below)
```suggestion
    debug(`Manifest for transformer ${transformer.name}`, options.manifest);
```

Reply:
got that at line 161

Reply:
I know, but if constructing the object fails, we won't have the transformer name logged. My request is to long earlier to make sure we have it.

Reply:
we don't have `transformer` until line 161. If `WorkerCallbackTransformer` constructor fails, we have the manifest logged.